crates/runtime/signalhandlers/SignalHandlers.cpp

```diff
@@ -399,41 +399,6 @@ struct AutoHandlingTrap
 
 }
 
-static
-#if defined(__GNUC__) || defined(__clang__)
-__attribute__ ((warn_unused_result))
-#endif
-bool
-HandleTrap(CONTEXT* context, bool reset_guard_page)
```

Comment:
Looks like there's a comment @ new line 404 that references `HandleTrap` as a function in this file (except it's now in Rust).